src/main/scala/scalachecklib/ArbitrarySection.scala

```diff
@@ -0,0 +1,96 @@
+package scalachecklib
+
+import org.scalatest.Matchers
+import org.scalatest.prop.Checkers
+
+/** ==The `arbitrary` Generator
+  *
+  * There is a special generator, `org.scalacheck.Arbitrary.arbitrary`, which generates arbitrary values of any
+  * supported type.
+  *
+  * {{{
+  * val evenInteger = Arbitrary.arbitrary[Int] suchThat (_ % 2 == 0)
+  * val squares = for {
+  *   xs <- Arbitrary.arbitrary[List[Int]]
+  * } yield xs.map(x => x*x)
+  * }}}
+  *
+  * The `arbitrary` generator is the generator used by ScalaCheck when it generates values for ''property'' parameters.
+  * Most of the times, you have to supply the type of the value to `arbitrary`, like above, since Scala often can't
+  * infer the type automatically. You can use `arbitrary` for any type that has an implicit `Arbitrary` instance.
+  * As mentioned earlier, ScalaCheck has default support for common types, but it is also possible to define your own
+  * implicit `Arbitrary` instances for unsupported types. See the following implicit Arbitrary definition for booleans,
+  * that comes from the ScalaCheck implementation.
+  *
+  * {{{
+  * implicit lazy val arbBool: Arbitrary[Boolean] = Arbitrary(oneOf(true, false))
+  * }}}
+  *
+  * @param name arbitrary
+  */
+object ArbitrarySection extends Checkers with Matchers with org.scalaexercises.definitions.Section {
+
+  import GeneratorsHelper._
+
+  /** Let's see an example where we're defining an `implicit` `arbitrary` instance for `Char`
+    */
+  def implicitArbitraryChar(res0: Seq[Char]) = {
+
+    import org.scalacheck.Arbitrary
+    import org.scalacheck.Gen
+    import org.scalacheck.Prop.forAll
+
+    implicit lazy val myCharArbitrary = Arbitrary(Gen.oneOf('A', 'E', 'I', 'O', 'U'))
+
+    val validChars: Seq[Char] = res0
```

Comment:
What's the purpose of `validChars`? To give a more specific meaning to `res0` val?